android_mobile/src/main/java/com/alexstyl/specialdates/dailyreminder/DailyReminderPreferences.kt

```diff
@@ -25,7 +25,7 @@ class DailyReminderPreferences(private val preferences: EasyPreferences)
     }
 
     override fun getRingtone(): URI {
-        val selectedRingtone = preferences.getString(R.string.key_daily_reminder_ringtone, null)
+        val selectedRingtone = preferences.getString(R.string.key_daily_reminder_ringtone, "")
```

Comment:
if you change the default value to empty string, the following line will never hit the default (because empty string is not null)